crates/wgsparkl3d/Cargo.toml

```diff
@@ -39,5 +39,47 @@ futures-test = "0.3"
 serial_test = "3"
 approx = "0.5"
 async-std = { version = "1", features = ["attributes"] }
-bevy = { version = "0.15.0", features = ["shader_format_glsl", "shader_format_spirv", "webgpu"] }
-wgsparkl_testbed3d = { path = "../wgsparkl-testbed3d" }
\ No newline at end of file
+bevy = { version = "0.15.0", features = [
+    "shader_format_glsl",
+    "shader_format_spirv",
+    "webgpu",
+    "jpeg",
+] }
+wgsparkl_testbed3d = { path = "../wgsparkl-testbed3d", features = ["dim3"] }
+obj-rs = { version = "0.7", default-features = false }
+bevy_editor_cam = "*"
+image = "0.25"
+gltf = { version = "1.3", features = ["KHR_materials_pbrSpecularGlossiness"] }
+
+[package.metadata.docs.rs]
+cargo-args = ["-Zunstable-options", "-Zrustdoc-scrape-examples"]
+
+[[example]]
+name = "banana3"
+path = "examples/banana3.rs"
+doc-scrape-examples = true
+crate-type = ["lib"]
+
+[[example]]
+name = "elastic_cut3"
+path = "examples/elastic_cut.rs"
+doc-scrape-examples = true
```

Comment:
migrate to a similar system than rapier: binaries rather than examples for better module reusability.